### PR TITLE
Add template / package / compile commands

### PR DESCRIPTION
**Pull request by williamjamir**

### CODE REVIEW

File: alfasim_sdk/cli.py [hunk 1]
```diff
@@ -1,15 +1,100 @@
...
+    type=click.Path(
+        exists=True,
+        file_okay=False,
+        writable=True,
+        resolve_path=True
+    ),
+    help='Path to the plugin directory')
+def compile(plugin_dir):
+    plugin_dir = Path(plugin_dir)
+    build_script = plugin_dir / 'build.py'
```
[the remark this answers is not in the document]
Comment (williamjamir) [reply alfasim_sdk/cli.py]: Currently, the hookman is calling this file as `build.py`.
I will change in both repositories from `build.py` to `compile.py`

File: alfasim_sdk/cli.py [hunk 2]
```diff
@@ -1,15 +1,100 @@
...
+@click.option('--plugin-dir',
+    default=os.getcwd(),
+    type=click.Path(
+        exists=True,
+        file_okay=False,
+        writable=True,
+        resolve_path=True
+    ),
+    help='Path to the plugin directory, where configuration and the shared library is located.')
+@click.option('--dst',
```
[the remark this answers is not in the document]
Comment (williamjamir) [reply alfasim_sdk/cli.py]: - [x] Fixed

File: tox.ini [hunk 3]
```diff
@@ -8,6 +8,7 @@ setenv =
 deps =
     -r{toxinidir}/requirements_dev.txt
     codecov
+    git+https://git@github.com/ESSS/hookman.git@create-template-command
```
[the remark this answers is not in the document]
Comment (williamjamir) [reply tox.ini]: Yep, I was just using to configure the CI

File: alfasim_sdk/cli.py [hunk 4]
```diff
@@ -1,15 +1,100 @@
...
+    ctx.invoke(compile, plugin_dir=plugin_dir)
+    plugin_dir = Path(plugin_dir)
+    dst = Path(dst)
+    hook_specs_file_path = _get_hook_specs_file_path()
+    hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
+    hm.generate_plugin_package(package_name, plugin_dir, dst)
+
+
+def _get_hook_specs_file_path():
+    import importlib
```
[the remark this answers is not in the document]
Comment (williamjamir) [reply alfasim_sdk/cli.py]: Nice call 😄

File: alfasim_sdk/cli.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamjamir) [reply alfasim_sdk/cli.py]: Changed ;)

**Comment by nicoddemus** [comment]
Good work!

**Pull request merged**